tests/TokenTest.cpp

```diff
@@ -562,6 +600,52 @@ TEST(TokenTest, VerifyTokenPS256FailNoKey) {
 	}(), jwt::rsa_exception);
 }
 
+#ifndef OPENSSL110
+TEST(TokenTest, VerifyTokenEd25519) {
+	const std::string token = "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXUyJ9.eyJpc3MiOiJhdXRoMCJ9.OujgVcO8xQx5xLcAYWENCRU1SCGH5HcX4MX4o6wU3M4"
+		"DOnKiNmc0O2AnvQlzr-9cgI4QGQzeC6gz_fgLoesADg";
+
+	auto verify = jwt::verify().allow_algorithm(jwt::algorithm::ed25519(ed25519_pub_key, "", "", ""));
+	auto decoded_token = jwt::decode(token);
+
+	verify.verify(decoded_token);
```

Comment:
please add at least one assertion

```suggestion
	ASSERT_NO_THROW(verify.verify(decoded_token));
```

Reply:
Would be okay by me, but no other success-tests have an `ASSERT_NO_THROW` right now, so it would not blend-in. Adding it to all success-tests would probably not fit the PR's topic very well. And gtest catches exceptions and fails the test without an ASSERT_NO_THROW, so it should not really change behaviour.